src/main/java/DataCenter/ChocAnServiceDriver.java

```diff
@@ -0,0 +1,17 @@
+package DataCenter;
+
+import Utils.Resources;
+
+public class ChocAnServiceDriver {
+
+    public static void main(String[] args)
+    {
+
+        //DataCenter.DataCenter chocAnCenter = new DataCenter.DataCenter();
+
+        Resources res = new Resources();
+        System.out.println(res.getUsername());
+        System.out.println(res.getPassword());
+        System.out.println(res.getConnection());
```

Comment:
Consider removing the print lines so that they don't go into master. The master branch should be maintained as close to perfect, working copy as possible.